bftengine/src/bftengine/ReplicasInfo.hpp

```diff
@@ -107,6 +108,7 @@ class ReplicasInfo {
   const std::set<PrincipalId> _idsOfExternalClients;
   const std::set<PrincipalId> _idsOfClientServices;
   const std::set<PrincipalId> _idsOfInternalClients;
+  const std::set<PrincipalId> _idsOfOperators;
```

Comment:
Let's keep also the order of initializations correct. _idsOfOperators should appear between _idsOfClientServices and _idsOfInternalClients.